### PR TITLE
Demo broken shareable configs

### PR DESCRIPTION
**Pull request by vdemedes**
PR to demonstrate https://github.com/sindresorhus/xo/issues/71#issuecomment-191247202. The test project is in `test/fixtures/project`.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Cool. Thanks for doing this. Could you make it something I could merge? So that tests can just fail until it's fixed.

**Comment by vdemedes** [comment]
You mean to `.skip()` the failing test?

File: package.json [hunk 1]
```diff
@@ -94,5 +94,8 @@
     "ignores": [
       "cli.js"
     ]
+  },
+  "ava": {
+    "files": "test/cli.js"
```
Comment (sindresorhus) [inline package.json]: I mean, remove this.

File: test/cli.js [hunk 2]
```diff
@@ -31,3 +31,9 @@ test.skip('ignores fixture', async t => {
 	const cwd = path.join(__dirname, 'fixtures/ignores');
 	t.throws(execa('../../../cli.js', ['--no-local'], {cwd}));
 });
+
+test.only('broken shareable config', async () => {
```
Comment (sindresorhus) [inline test/cli.js]: And make this just `test()`

Comment (sindresorhus) [inline test/cli.js]: And rename it to something like:

> supports shareable extended config

**Comment by vdemedes** [comment]
Done!

File: test/cli.js [hunk 3]
```diff
@@ -31,3 +31,9 @@ test.skip('ignores fixture', async t => {
 	const cwd = path.join(__dirname, 'fixtures/ignores');
 	t.throws(execa('../../../cli.js', ['--no-local'], {cwd}));
 });
+
+test('supports shareable config', async () => {
+	await execa('rm', ['-rf', '../node_modules/xo'], {cwd: __dirname});
```
Comment (sindresorhus) [inline test/cli.js]: Sorry, didn't notice this. Why is this needed?

Comment (vdemedes) [inline test/cli.js]: Oh yes, I guess this is another, separate issue. https://github.com/sindresorhus/xo/blob/master/package.json#L91 causes `cli.js` to stop execution and rely on `node_modules/xo` rather than version in `master` branch. So when you're testing `cli.js`, you are not actually testing it, because it spawns `cli.js` from `node_modules/xo`.

Comment (sindresorhus) [inline test/cli.js]: Hmm, that's what the `--no-local` flag is supposed to prevent, though.

Comment (vdemedes) [inline test/cli.js]: Oh ok, I'll change it.

Comment (vdemedes) [inline test/cli.js]: Done ;) Totally my ~~overshit~~ oversight.
